Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/UserInterface.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface UserInterface {
+
+	public String playerName();
+	
+	public void status();
+	
+	public boolean continuePrompt();
+	
+	public char guessPrompt();
+	
+	public void wrongGuess();
+	
+	public void rightGuess();
+	
+	public void repeatedGuess();
+	
+	public void winPrompt();
+	
+	public void losePrompt();
```

Comment:
I think all these methods could be more explicit in what they do from their name. Maybe something like, showLostPrompt() or showWrongGuessMessage(), right now these nouns don't actually say what the methods do.